README-zh.md

```diff
@@ -0,0 +1,181 @@
+This copy is called a *fork*. Make any changes you want in your fork,
+and when you are ready to send those changes to us, go to your fork and create a new pull request to let us know about it.
+ -->
+您可以点击屏幕右上方的 **Fork** 按钮，在您的 GitHub 账户下创建一份本仓库的副本。这个副本叫做 *fork*。您可以对 fork 副本进行任意修改，
+当准备好把变更提交给我们时，您可以通过创建一个 pull request 来通知我们。
+
+<!--
+Once your pull request is created, a Kubernetes reviewer will take responsibility for providing clear, actionable feedback.
+As the owner of the pull request, **it is your responsibility to modify your pull request to address the feedback that has been provided to you by the Kubernetes reviewer.**
+Also, note that you may end up having more than one Kubernetes reviewer provide you feedback or you may end up getting feedback from a Kubernetes reviewer that is different than the one initially assigned to provide you feedback.
+Furthermore, in some cases, one of your reviewers might ask for a technical review from a [Kubernetes tech reviewer](https://github.com/kubernetes/website/wiki/Tech-reviewers) when needed.
+Reviewers will do their best to provide feedback in a timely fashion but response time can vary based on circumstances.
+ -->
+创建 pull request 后，Kubernetes 审核人员将负责提供清晰且可操作的反馈。作为 pull request 的所有者，**您有责任修改 pull request 以解决 Kubernetes 审核者提供给您的反馈。**
+另请注意，您最终可能会收到多个 Kubernetes 审核人员为您提供的反馈，也可能出现后面 Kubernetes 审核人员的反馈与前面审核人员的反馈不尽相同的情况。
+此外，在某些情况下，您的某位评审员可能会在需要时要求 [Kubernetes 技术评审员](https://github.com/kubernetes/website/wiki/Tech-reviewers) 进行技术评审。
+审稿人将尽最大努力及时提供反馈，但响应时间可能因情况而异。
+
+<!--
+For more information about contributing to the Kubernetes documentation, see:
+
+* [Start contributing](https://kubernetes.io/docs/contribute/start/)
+* [Staging Your Documentation Changes](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
+* [Using Page Templates](http://kubernetes.io/docs/contribute/style/page-templates/)
+* [Documentation Style Guide](http://kubernetes.io/docs/contribute/style/style-guide/)
+* [Localizing Kubernetes Documentation](https://kubernetes.io/docs/contribute/localization/)
+ -->
+有关为 Kubernetes 文档做出贡献的更多信息，请参阅：
+
+* [开始贡献](https://kubernetes.io/docs/contribute/start/)
+* [缓存您的文档变更](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
+* [使用页面模版](http://kubernetes.io/docs/contribute/style/page-templates/)
+* [文档风格指南](http://kubernetes.io/docs/contribute/style/style-guide/)
+* [本地化 Kubernetes 文档](https://kubernetes.io/docs/contribute/localization/)
+
+<!--
+## `README.md`'s Localizing Kubernetes Documentation
+ -->
+## `README.md` 的本地化 Kubernetes 文档
+
+<!--
+### Korean
+
+You can reach the maintainers of Korean localization at:
+
+* June Yi ([GitHub - @gochist](https://github.com/gochist))
+* [Slack channel](https://kubernetes.slack.com/messages/kubernetes-docs-ko)
+ -->
+### 中文
+
+可以通过以下方式联系中文本地化的维护人员：
+
+* He Xiaolong ([GitHub - @markthink](https://github.com/markthink))
+* [Slack channel](https://kubernetes.slack.com/messages/kubernetes-docs-zh)
```

Comment:
you can also add my info in here too. 

`Rui Chen ([GitHub - @chenrui333](https://github.com/chenrui333))`

Reply:
Of course, I did think of that. 😄

Reply:
haha 👍